kong/clustering/rpc/manager.lua

```diff
@@ -141,6 +142,120 @@ function _M:_find_node_and_check_capability(node_id, cap)
 end
 
 
+-- CP => DP
+function _M:_handle_meta_call(c, node_id)
+  local data, typ, err = c:recv_frame()
+  if err then
+    return nil, err
+  end
+
+  assert(typ == "binary")
+
+  local payload = cjson_decode(data)
+  assert(payload.jsonrpc == "2.0")
+
+  if payload.method ~= RPC_MATA_V1 .. ".hello" then
```

Comment:
```suggestion
  if payload.method ~= RPC_MATA_V1 .. RPC_HELLO then
```

Reply:
I think it is not a must-do work, may refactor it later but not now.